tests/lib/reset.sh

```diff
@@ -1,20 +1,55 @@
 #!/bin/bash
 
-set -e -x
-
 # shellcheck source=tests/lib/dirs.sh
 . "$TESTSLIB/dirs.sh"
+
 # shellcheck source=tests/lib/state.sh
 . "$TESTSLIB/state.sh"
 
-
 # shellcheck source=tests/lib/systemd.sh
 . "$TESTSLIB/systemd.sh"
 
 #shellcheck source=tests/lib/systems.sh
-. "$TESTSLIB"/systems.sh
+. "$TESTSLIB/systems.sh"
+
+init_state_classic() {
+    if [ "$1" = "--reuse-core" ]; then
```

Comment:
Is `snapd{.service,.socket}` stopped at this point already?

Reply:
init state marged with clean as it is not needed anymore to keep them separated